hdmap/hdmap_shacl.ttl

```diff
@@ -38,7 +40,7 @@ hdmap:HdMapShape a sh:NodeShape ;
             sh:minCount 1 ;
             sh:node hdmap:GeoreferenceShape ;
             sh:order 7 ;
-            sh:path hdmap:georeference ] ;
+            sh:path georeference:georeference ] ;
```

Comment:
ti should be the other way round.
`sh:path` should be `hdmap:georeference`
`sh:node` should be `georeference:GeoreferenceShape`

Reply:
done